app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = t("base_title")
+    page_title.empty? ? base_title : "#{page_title} | #{base_title}"
```

Comment:
```suggestion
  def full_title page_title
    base_title = t "base_title"
    page_title.blank? ? base_title : "#{page_title} | #{base_title}"
```
Nên dùng blank em nhé
ko cần set giá trj default cho tham số nè